airbyte-cdk/java/airbyte-cdk/s3-destinations/src/test/java/io/airbyte/cdk/integrations/destination/s3/avro/AvroSerializedBufferTest.java

```diff
@@ -57,6 +58,8 @@ public static void setup() {
   }
 
   @Test
+  @Disabled("Flaky on CI, See run https://github.com/airbytehq/airbyte/actions/runs/7126781640/job/19405426141?pr=33201 " +
```

Comment:
thank you for adding a comment. Very useful for when someone decides to fix this